javascript/example/DRACOLoader.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
 
 THREE.DRACOLoader = function(manager) {
+    this.decoder = THREE.DRACOLoader.getDecoder();
```

Comment:
Oh, some clarification on the immediately invoked functions `(function(){})()` in JavaScript: both with and without this PR, the `DracoModule` is instantiated when the script loads — even before the `new THREE.DRACOLoader()` is created.

Happy to fix that while we're here, though. How does this look? I also switch from logging an error to throwing an error, since it's now happening at invocation and would throw an (less clear) error later regardless.